src/app/public/modules/file-attachment/file-drop.component.ts

```diff
@@ -30,33 +30,57 @@ import {
   styleUrls: ['./file-drop.component.scss']
 })
 export class SkyFileDropComponent implements OnDestroy {
+  /**
+   * Fires when users add or remove files.
+   */
   @Output()
   public filesChanged = new EventEmitter<SkyFileDropChange>();
 
   @Output()
   public linkInputBlur = new EventEmitter<void>();
 
+/**
+ * Fires when users add or remove links.
+ */
   @Output()
   public linkChanged = new EventEmitter<SkyFileLink>();
 
+/**
+ * Specifies the minimum size in bytes for valid files.
+ */
   @Input()
   public minFileSize: number = 0;
 
+/**
+ * Specifies the maximum size in bytes for valid files.
+ */
   @Input()
   public maxFileSize: number = 500000;
 
+/**
+ * Indicates whether users can drag and drop multiple files at the same time.
+ */
   @Input()
   public multiple: boolean = true;
 
+/**
+ * Specifies a custom validation function: `[validateFn]="validateFile"`.
+ */
   @Input()
   public validateFn: Function;
 
   @Input()
```

Comment:
Missing property description. It looks like we copied the description to the file attachment component instead of the file drop component. The file attachment component is not currently displayed in the docs, so I'm not sure if the description should be the same in both places.

Reply:
Fixed

Reply:
I'm still only seeing this description with `SkyFileAttachmentComponent` and not with `SkyFileDropComponent`, but based on the existing docs, it should be with both. (I was confused about that in my original comment, but looking at the docs again, I think both components need the same description for this property.)